__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.34'
```

Comment:
What does this do?  Seems like web/impact/impact/__init__.py is the important one.

Reply:
good catch. removed